website/client/src/components/notifications.vue

```diff
@@ -661,13 +661,7 @@ export default {
     showLevelUpNotifications (newlevel) {
       this.lvl();
       this.playSound('Level_Up');
-      // NOTE this code isn't actually used because no modal is shown when a quest is dropped
-      // In case it's added again it should keep in mind that it will not work
-      // when the user progress to the next level using the RYA modal
-      // as it doesn't score the tasks on the client side and thus this.user._tmp is not filled
-      // with any value
-      // if (this.user._tmp && this.user._tmp.drop && this.user._tmp.drop.type === 'Quest') return;
```

Comment:
I just realized that I added this comment in this PR some weeks ago https://github.com/HabitRPG/habitica/pull/11389/files#diff-b977df6ea07f466bb6bad7aa5d937e6bR648 and we should remove it since now quests drops (only happening on level ups) are shown in the level up modal and not in a separate one

Reply:
Indeed, that's why I deleted it here.

Reply:
Ah... I got confused 😅